static/canjs.js

```diff
@@ -9,6 +9,8 @@ var $articleContainer,
 
 // Init
 function init() {
+	document.title = window.docObject.title || window.docObject.name;
```

Comment:
I wonder if our titles should always have `canjs` in them if it isn't the homepage.  `CanJS - Guides` `canjs - can-component`, etc